src/main/java/net/sf/jabref/importer/fileformat/FieldContentParser.java

```diff
@@ -18,28 +18,44 @@
 import net.sf.jabref.gui.GUIGlobals;
 import net.sf.jabref.logic.util.strings.StringUtil;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * This class provides the reformatting needed when reading BibTeX fields formatted
  * in JabRef style. The reformatting must undo all formatting done by JabRef when
  * writing the same fields.
  */
 class FieldContentParser {
+    private final List<String> multiLineFields;
+
+    public FieldContentParser() {
+        multiLineFields = new ArrayList<>();
+        multiLineFields.add("abstract");
+        multiLineFields.add("review");
+    }
 
     /**
      * Performs the reformatting
-     * @param content StringBuffer containing the field to format. key contains field name according to field
-     *  was edited by Kuehn/Havalevich
-     * @return The formatted field content. NOTE: the StringBuffer returned may
-     * or may not be the same as the argument given.
+     *
+     * @param content StringBuffer containing the field to format. bibtexKey contains field name according to field
+     *                was edited by Kuehn/Havalevich
+     * @param bibtexKey
+     * @return The formatted field content. The StringBuffer returned may or may not be the same as the argument given.
      */
-    public StringBuffer format(StringBuffer content, String key) {
-        int i = 0;
-
+    public StringBuffer format(StringBuffer content, String bibtexKey) {
```

Comment:
I think the name `bibtexKey` is misleading, as I would think of it as the key of a bibtex entry. I would suggest to use `bibtexField` instead.